MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+module(name = "snmalloc")
+
+bazel_dep(name = "rules_cc", version = "0.1.1")
+bazel_dep(name = "rules_foreign_cc", version = "0.14.0")
+bazel_dep(name = "fuzztest", version = "20250214.0")
+bazel_dep(name = "googletest", version = "1.16.0")
+bazel_dep(name = "abseil-cpp", version = "20250127.1")
```

Comment:
I am not familiar with the dependency management, could you comment on why do we need to specify abseil-cop explicitly? I know fuzztest depends on it transitively.

Reply:
I added it explicitly for fuzz test. I should be able to remove it.